config/locales/hyrax.en.yml

```diff
@@ -31,6 +31,7 @@ en:
       carrierwave_processing_error: "Cannot resize image."
       extension_blacklist_error: "You are not allowed to upload %{extension} files, prohibited types: %{prohibited_types}"
       extension_whitelist_error: "You are not allowed to upload %{extension} files, allowed types: %{allowed_types}"
+      batch_delete_collections_error: "One or more of the selected collections could not be deleted. You cannot delete a collection for which you have read-only access. In addition, you cannot delete a collection of the type \"%{admin_set_name}\" unless that collection is empty."
```

Comment:
This key doesn't give much of a clue to where this message is used.  How about `hyrax.batch_edit.delete_collection.type_error`?

Reply:
yes!